ledger/src/blocktree_processor.rs

```diff
@@ -491,7 +493,8 @@ pub fn fill_blocktree_slot_with_ticks(
     parent_slot: u64,
     last_entry_hash: Hash,
 ) -> Hash {
-    let entries = create_ticks(ticks_per_slot, 0, last_entry_hash);
+    let num_slots = (slot - parent_slot).max(1); // Note: slot 0 has parent slot 0
```

Comment:
cooooo :D

Reply:
Yeah good call

Reply:
Following up in a small PR

Reply:
PR here: https://github.com/solana-labs/solana/pull/6664